app/views/payments/paymentHistory.scala.html

```diff
@@ -45,13 +45,22 @@ <h2>@model.selectedYear.toString</h2>
 
```

Comment:
I know this isn't a part of your subtask, but while you're here could you set the `allowPaymentHistory` feature to true in application.conf please. It's set to true in app-config-base and in Production so it should be true locally too.